### ISSUE TITLE
Incompatibility between modlauncher 8.1.x (mc 1.16) and Java 8 u321+

### ISSUE THREAD

**Issue by gigaherz** [McModLauncher/modlauncher#91]
The latest update to Java 8 applied a change to the `sun.security.util.ManifestEntryVerifier` class constructor, adding a parameter to it.

This breaks binary compatibility with modlauncher 8.1.x code:

https://github.com/McModLauncher/modlauncher/blob/3cf110cd0b2a5bcd8a44057fd0513e38a4b06f97/src/main/java/cpw/mods/modlauncher/SecureJarHandler.java#L66-L66

This change appears to also have been made to the other LTS branches (jdk11 and jdk17) as of last october:
- JDK 8: https://github.com/openjdk/jdk8u/commit/9edb3c788c590ca30b1bc94539035cf1e7c3c119
- JDK 11: https://github.com/openjdk/jdk11u/commit/9512c471e9ab0830d8d5b2275803f59a0f073fcb
- JDK 17: https://github.com/openjdk/jdk17u/commit/f89c59f9d151eea3f747155b129e17c9e6e8ec32

JDK 16 is not affected, and being EOL, will never be.

Stack trace (trimmed):
```
Exception in thread "main" [15:56:10] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:-1]: java.lang.NoSuchMethodError: sun.security.util.ManifestEntryVerifier.<init>(Ljava/util/jar/Manifest;)V
[15:56:10] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:-1]:         at cpw.mods.modlauncher.SecureJarHandler.createCodeSource(SecureJarHandler.java:66)
[15:56:10] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:-1]:         at cpw.mods.modlauncher.TransformingClassLoader$DelegatedClassLoader.findClass(TransformingClassLoader.java:275)
[15:56:10] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:-1]:         at cpw.mods.modlauncher.TransformingClassLoader.loadClass(TransformingClassLoader.java:136)
[15:56:10] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:-1]:         at cpw.mods.modlauncher.TransformingClassLoader.loadClass(TransformingClassLoader.java:98)
[15:56:10] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:-1]:         at java.lang.ClassLoader.loadClass(Unknown Source)
```

**Comment by usulthemouse** [comment]
what is the right version of java to downgrade too?

**Comment by SizableShrimp** [comment]
This is not a support forum; you should use the forge discord for further questions. However, anything below 8u321 should work fine. Or use the bundled JRE that came with the launcher.

**Comment by gigaherz** [comment]
Anything older than Java 8 u321, which would be Java 8 u312 or u311. 
Since Oracle's java archive requires an account, you can find it from the Adoptium project; https://adoptium.net/?variant=openjdk8&jvmVariant=hotspot

**Comment by sciwhiz12** [comment]
Fixed by e749b19a41f820c17bc0e4c3150d4e455d3337f2 (#92).

**Comment by huangsijun17** [comment]
Excuse me, which version of Forge started to fix this problem?

**Comment by jeremiahwinsley** [comment]
> Excuse me, which version of Forge started to fix this problem?

This was fixed in Forge 36.2.26.

**Comment by huangsijun17** [comment]
> > Excuse me, which version of Forge started to fix this problem?
> 
> This was fixed in Forge 36.2.26.

Thx.
